challenges/classes.js

```diff
@@ -1,7 +1,36 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+// function  CuboidMaker(props) {
+//     this.length = props.length;
+//     this.width = props.width;
+//     this.height = props.height
+//     this.volume = function () {
+//       return (props.length * props.width * props.height);
+//     };
+//     this.surfaceArea = function () {
+//       return 2 * (props.length * props.width + props.length * props.height + props.width * props.height);
+//     };
+//   }
+
+  class CuboidMaker {
+      constructor(length,width, height) {
+        this.length = length;
+        this.width = width;
+        this.height = height
+      }
+
+      volume() {
+        return (this.length * this.width * this.height)
```

Comment:
small note, I don't think you need the parens.  It will return after all the operators have finished unless you're using the `++` operator